core/startos/src/lib.rs

```diff
@@ -162,15 +172,26 @@ pub fn server<C: Context>() -> ParentHandler<C> {
                 .with_custom_display_fn(|handle, result| {
                     Ok(system::display_time(handle.params, result))
                 })
-                .with_call_remote::<CliContext>(),
+                .with_call_remote::<CliContext>()
+                .with_about("Display current time and server uptime"),
+        )
+        .subcommand(
+            "experimental",
+            system::experimental::<C>()
+                .with_about("Experimental commands such as zram and governor"),
```

Comment:
Maybe "Commands related to configuring experimental options such as zram and cpu governor"